wallet/lib/services/third_party_services/stripe_handler.dart

```diff
@@ -23,93 +21,93 @@ class StripeHandler {
-  /// @return StripeFailure, String Payout Transfer id
-  Future<Either<Failure, String>> handleStripePayout(String amount) async {
-    final dataSource = GetIt.I.get<LocalDataSource>();
-
-    final walletsStore = GetIt.I.get<WalletsStore>();
-
-    var token = '';
-    var accountId = '';
-
-    accountId = dataSource.getStripeAccountId();
-
-    if (accountId == "") {
-      final response = await repository.GenerateRegistrationToken(
-          walletsStore.getWallets().value.last.publicAddress);
-
-      if (response.isLeft()) {
-        return left(StripeFailure("stripe_registration_token_failed".tr()));
-      }
-
-      token = response
-          .getOrElse(() => StripeGenerateRegistrationTokenResponse())
-          .token;
-
-      if (token == "") {
-        return left(StripeFailure("stripe_registration_token_failed".tr()));
-      }
-
-      dataSource.saveStripeToken(token: token);
-
-      final register_response = await repository.RegisterAccount(
-          StripeRegisterAccountRequest(
-              Token: token,
-              Signature: await walletsStore.signPureMessage(token),
-              Address: walletsStore.getWallets().value.last.publicAddress));
-
-      if (register_response.isLeft()) {
-        return left(StripeFailure("stripe_register_account_failed".tr()));
-      }
-
-      final register_info =
-          register_response.getOrElse(() => StripeRegisterAccountResponse());
-
-      if (register_info.account == "" || register_info.accountlink == "") {
-        return left(StripeFailure("stripe_register_account_failed".tr()));
-      }
-
-      dataSource.saveStripeAccountId(accountId: register_info.account);
-    }
-
-    final payoutToken_response =
-        await repository.GeneratePayoutToken(StripeGeneratePayoutTokenRequest(
-      amount: Int64.parseInt(amount.ValToUval()),
-      address: walletsStore.getWallets().value.last.publicAddress,
-    ));
-
-    if (payoutToken_response.isLeft()) {
-      return left(StripeFailure("stripe_payout_token_failed".tr()));
-    }
-
-    final payoutToken = payoutToken_response
-        .getOrElse(() => StripeGeneratePayoutTokenResponse());
-
-    if (payoutToken.token == "") {
-      return left(StripeFailure("stripe_payout_token_failed".tr()));
-    }
-
-    final payout_response = await repository.Payout(StripePayoutRequest(
-      address: walletsStore.getWallets().value.last.publicAddress,
-      token: payoutToken.token,
-      amount: Int64.parseInt(amount.ValToUval()),
-      signature: await walletsStore.signPureMessage(payoutToken.token),
-    ));
-
-    if (payout_response.isLeft()) {
-      return left(StripeFailure("stripe_payout_request_failed".tr()));
-    }
-
-    final payout_info = payout_response.getOrElse(() => StripePayoutResponse());
-
-    if (payout_info.transfer_id == "") {
-      return left(StripeFailure("stripe_payout_request_failed".tr()));
-    }
-
-    return right(payout_info.transfer_id);
-  }
+  // /// handleStripePayout
```

Comment:
Remove dead code.

Reply:
But why we are removing stripe functionality?

Reply:
<img width="1241" alt="Screenshot 2022-09-14 at 10 03 54 AM" src="https://user-images.githubusercontent.com/106378847/190064109-01b9eacf-4364-4922-8322-811056577c0b.png">